app/frontend/styles/_primary-navigation.scss

```diff
@@ -2,30 +2,17 @@
   background-color: govuk-colour("light-grey");
 }
 
-.app-primary-navigation__container {
-  @include govuk-width-container;
-  font-size: 0; // Hide whitespace between elements
-  text-align: justify; // Trick to remove the need for floats
-
-  &:after {
-    content: '';
-    display: inline-block;
-    width: 100%;
-  }
```

Comment:
What's this for?

Reply:
@adamsilver thinks its related to the MoJ component which can contain a search field. We don’t have this, so isn’t needed. https://moj-design-system.herokuapp.com/components/primary-navigation/examples/inline-search